pages/builds/environment_variables.md.erb

```diff
@@ -4,51 +4,228 @@ When the agent invokes your build scripts it passes in a set of standard Buildki
+    <th><code>BUILDKITE_PIPELINE_PROVIDER</code></th>
+    <td>The ID of the source code provider for the pipeline’s repository. <p class="Docs__api-param-eg"><em>Example:</em> <code>"github"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PIPELINE_SLUG</code></th>
+    <td>The pipeline slug on Buildkite as used in URLs. <p class="Docs__api-param-eg"><em>Example:</em> <code>"my-project"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST</code></th>
+    <td>The number of the pull request if this branch is a pull request. <p class="Docs__api-param-eg"><em>Example:</em> <code>"123"</code> for pull request #123, or <code>"false"</code> if not a pull request.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST_BASE_BRANCH</code></th>
+    <td>The base branch that the pull request is targetting. <p class="Docs__api-param-eg"><em>Example:</em> <code>"master"</code>, or <code>""</code> if not a pull request.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST_REPO</code></th>
+    <td>The repository URL of the third-party pull request. <p class="Docs__api-param-eg"><em>Example:</em> <code>"git://github.com/acme-inc/my-project.git"</code>, or <code>""</code> if not a pull request.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_REBUILT_FROM_BUILD_ID</code></th>
+    <td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"4735ba57-80d0-46e2-8fa0-b28223a86586"</code>, or <code>""</code> if not a rebuild.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_REBUILT_FROM_BUILD_NUMBER</code></th>
+    <td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"1514"</code>, or <code>""</code> if not a rebuild.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_REPO</code></th>
+    <td>The repository of your pipeline. <p class="Docs__api-param-eg"><em>Example:</em> <code>"git@github.com:acme-inc/my-project.git"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_SOURCE</code></th>
+    <td>The source of the event that created the build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"webhook"</code>, <code>"api"</code>, <code>"ui"</code>, <code>"trigger_job"</code>, or <code>"schedule"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_TAG</code></th>
+    <td>The name of the tag being built, if this build was triggered from a tag. <p class="Docs__api-param-eg"><em>Example:</em> <code>"v1.2.3"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_TIMEOUT</code></th>
+    <td>The number of minutes until Buildkite automatically cancels this job, if a timeout has been specified. <p class="Docs__api-param-eg"><em>Example:</em> <code>"15"</code> for 15 minutes, or <code>"false"</code> if no timeout is set</td>
+  </tr>
+  <tr>
+    <th><code>CI</code></th>
+    <td>Always <code>"true"</code>.</td>
+  </tr>
+</tbody>
 </table>
 
+Bootstrap vars:
+BUILDKITE_DISABLE_GIT_SUBMODULES
```

Comment:
`BUILDKITE_DISABLE_GIT_SUBMODULES` can be supplied from the api, ui, or wherever triggers the build, or any hook before checkout to disable fetching any git submodules during checkout.